pkgs/tools/networking/burpsuite/default.nix

```
@@ -1,23 +1,26 @@
-{ stdenv, fetchurl, jre, runtimeShell }:
+{ stdenv, fetchurl, jre, writeShellScript, chromium }:
 
 let
-  version = "2020.1";
+  version = "2020.9.2";
+  embeddedChrome = "85.0.4183.121-1";
+  # Chrome version can be found using:  unzip -p burpsuite.jar chromium.properties
```

```suggestion
  # Chrome version can be found using:  unzip -p burpsuite.jar chromium.properties
  embeddedChrome = "85.0.4183.121-1";
```
Does this need a bump when updating?

---

Since you're already specifying a shell command to find the embedded chrome, couldn't we run this command as part of the package build phase to determine the correct chrome version automatically?